open_discussions/forms.py

```diff
@@ -0,0 +1,29 @@
+"""open_discussions forms"""
+from django import forms
+
+
+class LoginForm(forms.Form):
+    """Form to login an existing user"""
+    email = forms.EmailField(label='Email')
+    password = forms.CharField(label='Password', widget=forms.PasswordInput)
+
+
+class EmailForm(forms.Form):
+    """Form to start registration"""
+    email = forms.EmailField(label='Email')
+
+
+class PasswordAndProfileForm(forms.Form):
+    """Form to register a new user"""
+    fullname = forms.CharField(label='Full Name')
+    password = forms.CharField(label='Password', widget=forms.PasswordInput)
+    password_confirm = forms.CharField(label='Password Confirm', widget=forms.PasswordInput)
+
+    def clean(self):
+        """Validate that passowrd and confirmation match"""
+        cleaned_data = super().clean()
+        password = cleaned_data.get("password")
+        password_confirm = cleaned_data.get("password_confirm")
+
+        if password != password_confirm:
```

Comment:
I'm not sure there's a benefit of that here, because we're comparing two inputs from the same request, not user provided vs secret value which is where I think hmac comparisons are a necessity.

Reply:
Hmm, yeah you're right